.github/workflows/release.yml

```diff
@@ -240,6 +276,15 @@ jobs:
         cp "enarx-${{ matrix.architecture.build }}-unknown-linux-musl" dpkg/usr/bin/enarx
         chmod +x dpkg/usr/bin/enarx
         sudo chown root:root dpkg/usr/bin/enarx
+    - if: ${{ matrix.architecture.build == 'x86_64' }}
+      run: |
+        mkdir -p dpkg/usr/lib/enarx
+        cp "enarx-x86_64-unknown-linux-musl.sig" dpkg/usr/lib/enarx/enarx.sig
+        chmod 444 dpkg/usr/lib/enarx/enarx.sig
+        sudo chown root:root dpkg/usr/lib/enarx/enarx.sig
+    - run: |
```

Comment:
I don't think you want to do `dpkg -i` here yet, neither the `enarx --version`. Those should be run in the next step.